source/dyaml/loader.d

```diff
@@ -140,17 +138,18 @@ struct Loader
             return Loader(yamlData);
         }
         /// Ditto
-        private this(void[] yamlData) @system
+        private this(void[] yamlData, string name = "<unknown>") @system
         {
-            this(cast(ubyte[])yamlData);
+            this(cast(ubyte[])yamlData, name);
         }
         /// Ditto
-        private this(ubyte[] yamlData) @safe
+        private this(ubyte[] yamlData, string name = "<unknown>") @safe
```

Comment:
You've documented the new name parameter for Reader.this() as being an empty string if the buffer is just a string, but it's <unknown> here. Please choose one and correct the other.

Reply:
Done!
I also fixed the default value of `name` in `Reader.this` to make less confusion.